contents/blog/product-360.md

```diff
@@ -0,0 +1,124 @@
+---
+date: 2022-01-18
```

Comment:
Does this need to be updated to today's date?

Reply:
It does, yes. Thank you